index.html

```diff
@@ -128,6 +128,31 @@ <h4 class="p-heading-icon__title">Vanilla Design</h4>
   </div>
 </div>
 
+<div class="p-strip is-deep is-bordered">
```

Comment:
For future, this section should be pulled from the blog API instead of hard coding the latest articles.